src/libraries/Microsoft.Extensions.DependencyInjection/src/Microsoft.Extensions.DependencyInjection.csproj

```diff
@@ -4,6 +4,7 @@
     <TargetFrameworks>$(NetCoreAppCurrent);net461;netstandard2.0;netstandard2.1</TargetFrameworks>
     <!-- Debug IL generation -->
     <ILEmitBackendSaveAssemblies>False</ILEmitBackendSaveAssemblies>
+    <Nullable>Annotations</Nullable>
```

Comment:
are you planning to also enable warnings?

Reply:
The reason Annotation is enabled is because DI and Http use common shared files also used by Logging.Abstractions (NullExternalScopeProvider.cs and TypeNameHelper.cs) and they both have
```
#nullable enable
```
to them.